src/snowflake/cli/plugins/snowpark/commands.py

```diff
@@ -390,9 +411,10 @@ def build(
                         "Try again with --allow-shared-libraries."
                     )
             if download_result.packages_available_in_anaconda:
-                _write_requirements_file(
-                    snowpark_paths.snowflake_requirements_file,
-                    download_result.packages_available_in_anaconda,
+                _write_snowflake_requirements_file(
+                    file_path=snowpark_paths.snowflake_requirements_file,
+                    anaconda=anaconda,
```

Comment:
Same here, do we really need anaconda?

Reply:
Refactored - I moved this to AnacondaChannel